src/main/java/com/arextest/schedule/comparer/sqlparse/action/ActionFactory.java

```diff
@@ -0,0 +1,24 @@
+package com.arextest.schedule.comparer.sqlparse.action;
+import com.arextest.schedule.comparer.sqlparse.Parse;
+import net.sf.jsqlparser.statement.Statement;
+
+/**
+ * Created by rchen9 on 2023/1/6.
+ */
+public class ActionFactory {
+    public static Parse selectParse(Statement statement) {
+        String simpleName = statement.getClass().getSimpleName();
```

Comment:
>  String simpleName = statement.getClass().getSimpleName();
 don't use class simpleName for switch case, The name maybe change.

Reply:
done